core/mocks/generated/trezorconfig.pyi

```diff
@@ -1,6 +1,5 @@
 from typing import *
 
-
```

Comment:
as I said before, this whole directory should be excluded. in general, let's stick to what `make pystyle` produces and do not include any changes that are not touched by `make pystyle`